lib/danger_plugin.rb

```diff
@@ -234,8 +234,10 @@ def markdown_issues(results, heading)
         filename = r['file'].split('/').last
         line = r['line']
         reason = r['reason']
-
-        message << "#{filename} | #{line} | #{reason} \n"
+        rule = r['rule_id']
+        # Other available properties can be found int SwiftLint/…/JSONReporter.swift
+        message << "#{filename} | #{line} | #{reason} (#{rule})\n"
+        message << "#{filename} | #{line}"
```

Comment:
Wouldn't this code be adding two messages to the markdown sent back to the PR?

Reply:
ah, of course, this is duplicated by accident, thanks for spotting it!